tests/integration/ceph_mgr_test.go

```diff
@@ -121,20 +121,60 @@ func (suite *CephMgrSuite) execute(command []string) (error, string) {
 	return suite.cluster.installer.Execute("ceph", orchCommand, suite.namespace)
 }
 
+func (suite *CephMgrSuite) enableRookOrchestrator() (error) {
+	logger.Info("Enabling Rook orchestrator module: <ceph mgr module enable rook>")
+	err, output :=suite.cluster.installer.Execute("ceph", []string{"mgr", "module", "enable", "rook"}, suite.namespace)
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to enable rook orchestrator module: %q", err)
+		return err
+	}
+	logger.Info("Setting orchestrator backend to Rook .... <ceph orch set backend rook>")
+	err, output = suite.execute([]string{"set", "backend", "rook"})
+	logger.Infof("output: %s", output)
+	if err != nil {
+		logger.Infof("Not possible to set rook as backend orchestrator module: %q", err)
+	}
+	return err
+}
+
 func (suite *CephMgrSuite) waitForOrchestrationModule() {
 	var err error
+	var orchStatus map[string]string
+
+	err = suite.enableRookOrchestrator()
+	if err != nil {
+		logger.Error("First attemp: Error trying to set Rook orchestrator module")
+	}
+
 	for timeout := 0; timeout < 30; timeout++ {
+		logger.Info("Waiting for rook orchestrator module enabled and ready ...")
 		err, output := suite.execute([]string{"status"})
-		logger.Info("%s", output)
+		logger.Infof("output: %s", output)
 		if err == nil {
 			logger.Info("Rook Toolbox ready to execute commands")
+			// Convert string returned to map
+			outputLines := strings.Split(output, "\n")
+			orchStatus = make(map[string]string)
+			for _, setting := range outputLines {
+				s := strings.Split(setting, ":")
+				orchStatus[strings.TrimSpace(strings.ToLower(s[0]))] = strings.TrimSpace(strings.ToLower(s[1]))
+			}
+			if orchStatus["backend"] != "rook" {
+				err = suite.enableRookOrchestrator()
+				if err != nil {
+					continue
+				}
+			}
 			break
+		} else {
+			logger.Info("Rook orchestrator not ready. Enabling again ... ")
+			err = suite.enableRookOrchestrator()
 		}
 		time.Sleep(2 * time.Second)
 	}
-	logger.Error("Giving up waiting for Rook Toolbox to be ready")
-	assert.Nil(suite.T(), err)
```

Comment:
Don't we still need a check in case the module failed to be enabled?

Reply:
Not needed, if we cannot enable the rook module, all the test are going to fail.
Unless we should be able to cancel all the tests in a healthy way it really does not matter. (the time is going to be consumed trying to enable the rook module and if this is not possible, the suite continues and we have all the test failing in just a couple of seconds.)

I have tried to cancel the execution of the tests if we cannot enable the rook module .. but i not able to see a clean way to cancel the tests ad continue executing the teardown in order to clean the test environment. 
In any case.. we do not obtain a big benefit doing that.

Reply:
Actually, a clean way to terminate the test should be to change the assert to require:
`require.Nil(suite.T(), err)`

This way, the remaining tests won't fail unexpectedly and we should have a better error message to indicate the root cause.